app/src/main/java/com/door43/translationstudio/tasks/UpdateAppTask.java

```diff
@@ -149,6 +147,8 @@ private void updateBuildNumbers() {
      */
     private void upgradePre115() {
         AppContext.context().deleteDatabase(Library.DATABASE_NAME);
+        TargetTranslation[] targetTranslations = AppContext.getTranslator().getTargetTranslations();
+        TargetTranslationMigrator.migrateFromUSXtoUSFM(targetTranslations);
```

Comment:
@neutrinog - also it looks like TargetTranslationMigrator.migrateChunkChanges() is only called on upgradePre110().  It seems like you would want to call this whenever the app version is changed, or maybe whenever the library is updated because chunks could change.